### ISSUE TITLE
Split trainset in train and validation split

### ISSUE THREAD

**Issue by chriskamphuis** [Chriskamphuis/ASR#13]
We have to look at the test set to know how to create the validation set, the validation set should be a good representation of how the test should behaves (e.g. split per sentence or per speaker)

**Comment by TimovNiedek** [comment]
See TIMIT RNN.ipynb
